### ISSUE TITLE
CART loss not used in default SFT config

### ISSUE THREAD

**Issue by Peterande** [DreamLM/Dream#62]
@zhxieml Nice work! According to https://github.com/DreamLM/Dream/blob/fd91b8f1d47c5cbe4a8a1674fd9b98045e79d9db/src/trainer/config/sft_trainer.yaml#L56-L57, the default SFT hyperparameters do not use CART loss. Could I kindly ask the author what values of `cart_p` were used during the PT (pretraining) and SFT (supervised fine-tuning) stages of Dream? Thank you very much!
